src/components/Button/ButtonGroup.tsx

```diff
@@ -0,0 +1,25 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as Rivet from '../Rivet';
+
+interface ButtonGroupProps {
+  /**
+   * Right align the buttons in this button group
+   * @see https://rivet.uits.iu.edu/components/forms/buttons/#button-groups
+   */
+  right?: boolean;
+};
+
+const ButtonGroup: React.SFC<ButtonGroupProps & React.HTMLAttributes<HTMLDivElement>> = ({ className, children, right, ...attrs }) => {
+  const classes = classNames({
+    ['rvt-button-group']: true,
```

Comment:
I just thought of this, but are the square brackets required here?

Reply:
No, they are not but I was including them explicitly so it would be easier to read since the keys would all be written the same way.